Library/Homebrew/test/cmd/upgrade_spec.rb

```diff
@@ -16,4 +16,17 @@
     expect(HOMEBREW_CELLAR/"testball/0.1").to be_a_directory
     expect(HOMEBREW_CELLAR/"testball/0.0.1").not_to exist
   end
+
+  it "can do a dry run upgrade" do
```

Comment:
Remove this test; don't want to add new (slow) integration tests for flags, thanks.